Draft.md

```diff
@@ -168,159 +168,201 @@ Im folgenden stellen wir einen kommentierten Fragenkatalog mit Kriterien zur Beu
+* **Ist die Dokumentation aktuell und adressiert sie alle Funktionen des Programms?** Das letzte Bearbeitungsdatum einer Dokumentation ist meist einem Zeitstempel zu entnehmen, wenn die Dokumentation mit einem Dokumentationswerkzeug, wie oben genannt, erstellt wurde. Dieses Datum sollte mit dem Veröffentlichungsdatum der aktuellsten Software übereinstimmen oder aktueller sein. Werden im besten Fall die Prinzipien der *Continuous Integration* angewendet, so ist die Erzeugung und Bereitstellung der Dokumentation direkt mit im Entwicklungsprozess der Anwendung integriert, wie beispielsweise das [GitHub Repository des SPARQLing Unicorn QGIS Plugins](https://github.com/sparqlunicorn/sparqlunicornGoesGIS). Hier erstellt ein automatisierter Prozess die Dokumentation bei jeder Veränderung des Quellcodes neu und publiziert sie auf der [GitHub Page des Repositories](https://sparqlunicorn.github.io/sparqlunicornGoesGIS/).
 
-* **Ist der Buildprozess dokumentiert und ggf. mittels Buildingscripts automatisiert?** Ebenso wichtig wie ein grundlegendes Verständnis des Aufbaus des Programms ist die Bauanleitung der Software. Entsprechend ist zu fragen, ob dokumentiert ist, wie die Software erstellt wurde und funktioniert die Erstellung unter dem angegebenen Weg. Während in der Vergangenheit mit Anleitungen in *README Files* oder ähnlichen natürlichsprachlichen Beschreibungen gearbeitet wurde, hat es sich seit vielen Jahren zu einem Standard etabliert, maschinenlesbare Bauanleitungen (*Buildscripts*) für Software bereitzustellen. Dies kann als gute Praxis angesehen werden, denn zum einen ist eindeutig und maschinenlesbar beschrieben, wie die Software erstellt wurde, zum anderen ist der Aufruf des Buildprozess so meist durch Ausführung eines einzigen Skriptes möglich und somit für den Entwickler einfach gehalten. Ebenfalls dokumentieren die Buildscripts die Abhängigkeiten der Software in Bezug auf verwendete *Libraries*. Beispiele für solche Skripts finden sich z.B. in [Apache Maven](https://maven.apache.org) oder [Gradle](https://gradle.org)
+> [MT] Da CI hier schon genannt ist, habe ich die Erläuterung hier er verschoben. Man könnte aber auch überlegen das als eigene Frage in diesem Bereich aufzugreifen. Oder, wie ganz unten angemerkt, allen Aspekten in der Entwcklerperspektive voran zu stellen.
```

Comment:
Am besten würde es imho in die Einleitung zu `#### Dokumentation und Tests` passen.

Reply:
Habe das mal probiert. Müsste aber nochmal gegengelesen werden